pkg/limayaml/defaults.go

```diff
@@ -121,22 +121,28 @@ func FillDefault(y, d, o *LimaYAML, filePath string) {
 			}
 		}
 	}
+	yamlOverride := false
```

Comment:
Is this variable needed?

Reply:
Yes yes, for cpuType we are setting defaults first and overriding based on yaml property. 

I went with this variable to avoid doing append(y, d, o) and get length from there.